controllers/workloads/integration/cfbuild_controller_integration_test.go

```diff
@@ -264,3 +265,10 @@ var _ = Describe("CFBuildReconciler Integration Tests", func() {
 		})
 	})
 })
+
+func setKpackImageStatus(kpackImage *buildv1alpha1.Image, conditionType string, conditionStatus string) {
```

Comment:
Where did this come from? This wasn't removed from elsewhere and no usages of it seem to have been added.

Reply:
It gets used in both the unit and integration tests, but I didn't want to add it to the sharedtest utils because other tests are depending on that package too.